engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql

```diff
@@ -1569,4 +1569,12 @@ CREATE VIEW `cloud`.`user_view` AS
             left join
         `cloud`.`async_job` ON async_job.instance_id = user.id
             and async_job.instance_type = 'User'
-            and async_job.job_status = 0;
\ No newline at end of file
+            and async_job.job_status = 0;
+
+-- Change usage of VM_DISK_IO_WRITE to use right usage_type
+UPDATE
+  `cloud_usage`.`cloud_usage`
+SET
+  usage_type = 22
+WHERE
```

Comment:
This might take very long time if usage database is huge. (It is quite normal if usage server has heen running for some years).

Might it be helpful if use the following ?

WHERE usage_type = <type of UsageTypes.VM_DISK_BYTES_WRITE> AND usage_display like '% io write';

Reply:
@weizhouapache done, thanks for your suggestion.